src/components/HeroHeader/index.tsx

```diff
@@ -8,6 +8,8 @@ interface IHeroHeader {
   img: ReactNode;
   imgStyle?: React.CSSProperties;
   imgAlt: string;
+  titleStyle?: React.CSSProperties; // Added titleStyle prop
+  subtitleStyle?: React.CSSProperties; // Added subtitleStyle prop
```

Comment:
are these comment are necessary?
```suggestion
  titleStyle?: React.CSSProperties;
  subtitleStyle?: React.CSSProperties;
```